packages/dbos-compiler/templates/method.create.liquid

```diff
@@ -0,0 +1,69 @@
+CREATE FUNCTION "{{appVersion}}{{methodName}}_func"(
```

Comment:
Does it mean the methodName has to be globally unique? What happens if two classes have the same methodName?

Reply:
I thought we already had a requirement that DBOS WF/Comm/Tx names had to be unique

Reply:
I will add class name to the function name

Reply:
added class name to generated routine names + added check to compiler to ensure class + method name length doesn't exceed 48 characters. Routines need 12 characters for version prefix and 2 characters for type suffix (`_f` or `_p`) and 1 character underscore between class & method names. [Maximum identifier length in PG is 63 characters](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#:~:text=The%20system%20uses%20no%20more,identifier%20length%20is%2063%20bytes), 

63 char max - 15 char overhead = 48 char class + method name max